.npmignore

```diff
@@ -0,0 +1,5 @@
+node_modules/
```

Comment:
Note that the `.npmignore` file is _independent_ of the `.gitignore` file and _replaces_ the latter as far as npm is concerned.

This file must contain anything that is not to be published.